src/audits/PageWithBadTitle.js

```diff
@@ -0,0 +1,43 @@
+// Copyright 2015 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+goog.require('axs.AuditRules');
+goog.require('axs.constants.Severity');
+
+axs.AuditRules.addRule({
+    name: 'pageWithBadTitle',
+    heading: 'The web page should have a title less than 66 characters long and does not contain bad characters',
```

Comment:
@kristapsmelderis can you be more specific about the genesis of this rule? Is there a corresponding standard that drove this out? Why 66 characters?

I'm hesistant to merge this unless there's a convention or standard backing it up.